Entities/Industry/CTFShops/BoatShop/BoatShop.as

```diff
@@ -57,5 +57,21 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 	if (cmd == this.getCommandID("shop made item"))
 	{
 		this.getSprite().PlaySound("/ChaChing.ogg");
+		bool isServer = (getNet().isServer());
+		u16 caller, item;
+		if (!params.saferead_netid(caller) || !params.saferead_netid(item))
+		{
+			return;
+		}
+		string name = params.read_string();
```

Comment:
saferead should be used as you did previously.

Reply:
I was just copypasting code from VehicleShop.as so that it looks the same. I've changed all cases of "shop made item" command in all CTF shops now so that they all use saferead.

(also changed getNet().isServer() to isServer() in the files)